.github/workflows/runtestapp.yml

```diff
@@ -0,0 +1,25 @@
+name: RunTests
+
+on: [push, pull_request]
+
+jobs:
+  test-py3:
+    container:
+      image: nvcr.io/nvidia/pytorch:19.10-py3
```

Comment:
container:
      image: nvcr.io/nvidia/pytorch:19.10-py3
      options: --user 1001

Reply:
looks like actions/checkout@v2 works fine now, should I still add the option?

Reply:
If v2 solves it, then no need for that user id setting.  However, I may need to move the entire runner to a different folder to avoid showing my home directory.